contrib/add_license_to_files.jl

```diff
@@ -58,7 +58,7 @@ const skipfiles = [
     "../src/support/tzfile.h",
     "../src/support/utf8.c",
     "../test/perf/micro/randmtzig.c",
-    "../src/support/crc32c.c",
+    "../src/crc32c.c",
```

Comment:
No. The commit changes the file enough that git may not show them as the same file. Also there isn't anything different between `git mv` and remove + add in general.